newIDE/app/src/ProjectManager/index.js

```diff
@@ -299,6 +306,19 @@ export default class ProjectManager extends React.Component<Props, State> {
     }
   }
 
+  _onEditLayoutVariable = (name: any = null) => {
```

Comment:
Never use any, don't use default parameters

Reply:
I used any because the name variable could be of two types (null ie object and string)